README.md

```diff
@@ -1,226 +1,31 @@
 
 ```sh
-docker build -t avalanche:protobuf_codegen -f api/Dockerfile.buf .
-docker run -t -i -v $(pwd):/opt/avalanche -w/opt/avalanche avalanche:protobuf_codegen bash -c "scripts/protobuf_codegen.sh"
+docker pull step/step-node
+docker run -v /step-node-db:/root/.avalanchego/db \
+  --network=host \
+  --name step-node step/step-node:latest avalanchego \
+  --config-file=/root/.avalanchego/config.json \
+  --chain-config-dir=/root/.avalanchego/configs/chains-restricted \
+  --public-ip=<your.public.ip.address>
 ```
 
-## Supported Platforms
-
-AvalancheGo can run on different platforms, with different support tiers:
-
-- **Tier 1**: Fully supported by the maintainers, guaranteed to pass all tests including e2e and stress tests.
-- **Tier 2**: Passes all unit and integration tests but not necessarily e2e tests.
-- **Tier 3**: Builds but lightly tested (or not), considered _experimental_.
-- **Not supported**: May not build and not tested, considered _unsafe_. To be supported in the future.
-
-The following table lists currently supported platforms and their corresponding
-AvalancheGo support tiers:
-
-| Architecture | Operating system | Support tier  |
-| :----------: | :--------------: | :-----------: |
-|    amd64     |      Linux       |       1       |
-|    arm64     |      Linux       |       2       |
-|    amd64     |      Darwin      |       2       |
-|    amd64     |     Windows      |       3       |
-|     arm      |      Linux       | Not supported |
-|     i386     |      Linux       | Not supported |
-|    arm64     |      Darwin      | Not supported |
-
-To officially support a new platform, one must satisfy the following requirements:
-
-| AvalancheGo continuous integration | Tier 1  | Tier 2  | Tier 3  |
-| ---------------------------------- | :-----: | :-----: | :-----: |
-| Build passes                       | &check; | &check; | &check; |
-| Unit and integration tests pass    | &check; | &check; |         |
-| End-to-end and stress tests pass   | &check; |         |         |
+## Becoming Step Validator
 
-## Security Bugs
+As well as the main Avalanche networks (X-, P- and C-), Step Network is secured by Avalanche chain validators explicitly configured to validate the subnet. To become step validator you should become validator on Avalanche network (Fuji Testnet).
 
-**We and our community welcome responsible disclosures.**
+Follow [this guide](https://docs.avax.network/nodes/validate/add-a-validator#fuji-workflow) and execute following steps:
 
-If you've discovered a security vulnerability, please report it via our [bug bounty program](https://hackenproof.com/avalanche/). Valid reports will be eligible for a reward (terms and conditions apply).
+* Run your node as described in previous paragraph
+* Save its ID and export staker.crt and staker.key generated by the node (recommended way is to mount staking folder as volume with using -v argument. Example: `-v /avalanchego/staking:/root/.avalanchego/staking`).
```

Comment:
```suggestion
* Save its ID and export staker.crt and staker.key generated by the node (recommended way is to mount staking folder as volume with using -v argument. Example: `-v /avalanchego/staking:/root/.avalanchego/staking`).
```
* Save its ID and export staker.crt and staker.key generated by the node (recommended way is to mount staking folder as volume with using -v argument. Example: 
```
docker pull step/step-node
docker run -v /step-node-db:/root/.avalanchego/db -v /avalanchego/staking:/root/.avalanchego/staking \
  --network=host \
  --name step-node step/step-node:latest avalanchego \
  --config-file=/root/.avalanchego/config.json \
  --chain-config-dir=/root/.avalanchego/configs/chains-restricted \
  --public-ip=<your.public.ip.address>
```